### PR TITLE
MP-00/Take_API_Key_for_BigOven_out_of_Java

### PR DESCRIPTION
**Pull request by Mulderink1**
Created constant folders and .gitignored the keys constant class. I also added an additional Constants folder for reused variables. I also added the build mv to the .gitignore.

### CODE REVIEW

File: recipe_box/src/main/java/com/spr/recipe_box/RecipeService.java [hunk 1]
```diff
@@ -17,7 +18,7 @@
     private static List<Recipe> recipeList = new ArrayList<>();
 
     public List<Recipe> findById(String recipe_id){
-            String url  = "https://api.bigoven.com/recipe/"+recipe_id+"?api_key=glFUKikehWjLW900etpS564VgIzOWSW5";
+            String url  = "https://api.bigoven.com/recipe/"+recipe_id+ Keys.BIG_OVEN_KEY;
```
Comment (emritcey) [inline recipe_box/src/main/java/com/spr/recipe_box/RecipeService.java]: Can we change this to where: 
BIG_OVEN_KEY = glFUKikehWjLW900etpS564VgIzOWSW5
and keep the  `?api_key` out here?

Comment (emritcey) [reply recipe_box/src/main/java/com/spr/recipe_box/RecipeService.java]: Also, if we are changing the end points to constants can we add one for  `https://api.bigoven.com/`?

File: recipe_box/src/main/java/com/spr/recipe_box/RecipeDBRestController.java [hunk 2]
```diff
@@ -15,32 +15,33 @@
 @RequestMapping(value = "/recipes", produces = "application/json")
 
 public class RecipeDBRestController extends RestClass {
+    private static final String RECIPES = "/recipes/";
```
Comment (emritcey) [inline recipe_box/src/main/java/com/spr/recipe_box/RecipeDBRestController.java]: This is soooo nitpicky, but any reason this has to be all caps?

**Review by emritcey — APPROVED** [review]

File: recipe_box/src/main/java/com/spr/recipe_box/RecipeDBRestController.java (hunk 2 above)
Comment (Mulderink1) [reply recipe_box/src/main/java/com/spr/recipe_box/RecipeDBRestController.java]: It's the general naming convention for Constants in Java -> https://howtodoinjava.com/java/basics/java-naming-conventions/

File: recipe_box/src/main/java/com/spr/recipe_box/RecipeService.java (hunk 1 above)
Comment (Mulderink1) [reply recipe_box/src/main/java/com/spr/recipe_box/RecipeService.java]: I completely agree I'll get this switched over!

File: recipe_box/src/main/java/com/spr/recipe_box/RecipeDBRestController.java (hunk 2 above)
Comment (emritcey) [reply recipe_box/src/main/java/com/spr/recipe_box/RecipeDBRestController.java]: great resource thanks for sharing !

**Pull request merged**